readthedocs/proxito/urls.py

```diff
@@ -160,26 +160,36 @@
 dummy_dashboard_urls = [
     # /dashboard/<project_slug>/
```

Comment:
I guess the comments need to be updated.

```suggestion
    # /projects/<project_slug>/
```

Reply:
Thanks. I changed them all.